### DISCUSSION TITLE
seurat object and CallPeaks

### DISCUSSION THREAD

**Discussion by shinlin77** [stuart-lab/signac#884] [Q&A] [answered]
I have to read a lot of multiome files into RAM. and create a seurat object  It takes about a day.  If I go straight through to CallPeaks, it works fine.  However, if I save it as a *h5seurat file, and read it in then I get the following problem.

peaks  = CallPeaks(object, macs2.path = "/usr/bin/macs2", group.by = "cell_type")
Processing file /mydir/outs/atac_fragments_rename.tsv.gz
Keeping 0 cell barcodes

Somehow, it can't find the cell barcodes.  Can this be fixed?

**Comment by timoast** [comment, marked as answer]
This seems to be due to an issue with SeuratDisk, I opened an issue here: https://github.com/mojaveazure/seurat-disk/issues/97
